### PR TITLE
Use KVM on tests that run on self-hosted runners

### PR DESCRIPTION
**Pull request by jimmykarily**
to make the run faster

Part of #771

### CODE REVIEW

File: .github/workflows/image.yaml [hunk 1]
```diff
@@ -210,13 +210,35 @@ jobs:
...
+            global:
+              buildkit_additional_config: |
+                [registry."docker.io"]
+                  mirrors = ["registry.docker-mirror.svc.cluster.local:5000"]
+                [registry."registry.docker-mirror.svc.cluster.local:5000"]
+                  insecure = true
+            EOF
+
+            earthly.sh +prepare-bundles-tests
+            earthly.sh +run-qemu-bundles-tests --FLAVOR=${{ matrix.flavor }}
```
Comment (mauromorales) [inline .github/workflows/image.yaml:241]: shouldn't these be just `earthly` without the script?

Comment (jimmykarily) [reply .github/workflows/image.yaml:242]: oops, yes!

**Review by mauromorales — APPROVED** [review]
nice :sunglasses:

**Comment by jimmykarily** [comment]
KVM is not used when using earthly. This needs some more work to run tests without earthly.

**Pull request merged**